### PR TITLE
Feature/java11 neo4jv4

### PR DESCRIPTION
**Pull request by DirkMahler**

### CODE REVIEW

File: status.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Java 11
```
Comment (StephanPirnbaum) [inline status.txt:1]: Please remove this file

File: store/src/main/java/com/buschmais/jqassistant/core/store/impl/AbstractGraphStore.java [hunk 2]
```diff
@@ -58,7 +58,7 @@ public void start() {
             .classLoader(ofNullable(storePluginRepository.getClassLoader()))
             .types(storePluginRepository.getDescriptorTypes())
             .validationMode(ValidationMode.NONE)
-            .clearAfterCompletion(false)
+     //       .clearAfterCompletion(false)
```
Comment (StephanPirnbaum) [inline store/src/main/java/com/buschmais/jqassistant/core/store/impl/AbstractGraphStore.java:61]: ?

**Review by StephanPirnbaum — CHANGES_REQUESTED** [review]

File: status.txt (hunk 1 above)
Comment (DirkMahler) [reply status.txt]: Removed

File: store/src/main/java/com/buschmais/jqassistant/core/store/impl/AbstractGraphStore.java (hunk 2 above)
Comment (DirkMahler) [reply store/src/main/java/com/buschmais/jqassistant/core/store/impl/AbstractGraphStore.java]: Change required for debugging and accidentially committed . Now reverted.

**Review by StephanPirnbaum — APPROVED** [review]

**Review by obfischer — APPROVED** [review]

**Pull request merged**